src/utils/parse.test.js

```diff
@@ -1,26 +1,20 @@
-import _ from 'lodash';
 import { generateSampleTable, generateClusters } from './parse';
-import { mockDataSample, expectedSampleData, expectedClusterData } from '../../mock/api';
+import { mockSamples } from '../../mock/api';
+
+const parsedSampleTable = generateSampleTable([mockSamples]);
+const parsedClusterData = generateClusters(
+  parsedSampleTable.runs,
+  parsedSampleTable.iterationParams
+);
 
 describe('test generateSampleTable', () => {
-  const parsedSampleTable = generateSampleTable(mockDataSample);
   it('should generate sample table data', () => {
     expect(Object.keys(parsedSampleTable).length > 0);
   });
-  it('should equal mocked sample table data', () => {
-    expect(_.isEqual(parsedSampleTable, expectedSampleData));
-  });
 });
 
 describe('test generateClusters', () => {
-  const parsedClusterData = generateClusters(
-    expectedSampleData.runs,
-    expectedSampleData.iterationParams
-  );
   it('should generate cluster data', () => {
     expect(Object.keys(parsedClusterData).length > 0);
   });
-  it('should equal mocked cluster data', () => {
```

Comment:
Please note we'll want to consider a new way for comparing mocked outcomes from our postprocessing without having the massive parsed response defined in our mock files. Eventually, this definition will move to the server as the server will handle postprocessing before sending a simplified response to the dashboard.